### DISCUSSION TITLE
How to report a security issue?

### DISCUSSION THREAD

**Discussion by piergiorgioladisa** [harfbuzz/harfbuzz#3611] [Q&A] [answered]
As described in the title, what is your designed channel to report a security issue?

**Comment by behdad** [comment, marked as answer]
Please email me behdad behdad org.

**Reply by piergiorgioladisa** [reply]
Hi @behdad I sent you an email last friday with the bug report but I didn't received any answer nor acknowledgment. Due to the criticality of the issue I just wanted to be sure that you correctly received my email.

**Reply by behdad** [reply]
Hi @piergiorgioladisa
I've actually been looking forward to your email. But I'm afraid I didn't receive it. Can you send again please?

**Reply by piergiorgioladisa** [reply]
@behdad I've just sent you an email containing the previous email.

**Reply by behdad** [reply]
@piergiorgioladisa Weird. I still didn't receive it. Does it have payload that maybe GMail filters are triggered by? Can you send an innocent email to establish communication?

**Reply by behdad** [reply]
Obviously, I checked spam folder as well.

**Reply by piergiorgioladisa** [reply]
@behdad email sent just with a small text :)

**Reply by behdad** [reply]
@piergiorgioladisa Just to make sure, to behdad.org, not accidentally behdad.com, right?

**Reply by behdad** [reply]
Alternatively, it's my first.last at gmail. Still have not received it.

**Reply by piergiorgioladisa** [reply]
@behdad yes I just checked the email is correct

**Reply by behdad** [reply]
@piergiorgioladisa Okay got the test email now.
